assets/src/components/notifications.tsx

```diff
@@ -6,10 +6,17 @@ import { Notification, NotificationReason } from "../realtime.d"
 import { formattedTimeDiff, now } from "../util/dateTime"
 import PropertiesList from "./propertiesList"
 
+const fullstoryEvent = (): void => {
```

Comment:
@skyqrose I would suggest naming this `deliveryFullstoryEvent`. That makes it clear at a glance what circumstances trigger this event (notification delivered, i.e. successfully rendered).